app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,41 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <% administrations.each do |title, field_name| %>
+        <%= f.fields_for field_name do |f| %>
+          <p class="h4"><%= title %></p>
+          <div class="form-group">
+            <div class="form-wrapper radio">
```

Comment:
```suggestion
            <div class="radio">
```

I don't think these form-wrapper classes do anything for us here and they seem inconsistently applied compared to how they're applied on other elements (they seem to be use to wrap a control where label is a separate element).

Same for the other `.form-wrapper .radio` selectors.

Reply:
No, I don't think `form-wrapper` adds much here. This was an attempt to be consistent with [this instance](https://github.com/alphagov/publisher/blob/3e083c9998692847cbb7325c0e9234684dc1be35/app/views/shared/_common_edition_attributes.html.erb#L21). Have removed `form-wrapper`. But, we may need/want to revisit these classes for consistency when dealing with the error handling.